flink-tests/src/test/java/org/apache/flink/runtime/operators/lifecycle/PartiallyFinishedSourcesITCase.java

```diff
@@ -103,7 +103,7 @@ private TestJobWithDescription buildJob() {
                     env.setRestartStrategy(fixedDelayRestart(1, 0));
                     // checkpoints can hang (because of not yet fixed bugs and triggering
                     // checkpoint while the source finishes), so let them timeout quickly
```

Comment:
Let's update the comment while merging. I think it is no longer the goal to timeout them quickly.